### PR TITLE
Harrison react navigation

### LINKED ISSUE #22: add react native navigation
no specific frames. needed for overall functionality.

### PR DESCRIPTION
**Pull request by hpweisberg**
Added react-navigation. Add sample tailwind css to mainBtn. Create a navigation example to the login page when clicking signup/in

### CODE REVIEW

File: App.tsx [hunk 1]
```diff
@@ -119,4 +103,36 @@ function App(): JSX.Element {
 //   },
 // });
 
+// ! Not in use. We can only have Stack.Screen inside nvigator:
```
Comment (ekelly55) [inline App.tsx:106]: Are you intending to leave this commented out section in the file? I see that it's new, but also commented out.

File: ios/Podfile.lock [hunk 2]
```diff
@@ -686,6 +701,7 @@ SPEC CHECKSUMS:
   React-jsiexecutor: 990287d74aedc4fdd08ebd80736b1a5c71b54da2
   React-jsinspector: 8d754fc957255a29d93e52fc67a895045cdc8703
   React-logger: 454ffb01980778a43b0153ee98721d0275b56616
+  react-native-safe-area-context: 68b07eabfb0d14547d36f6929c0e98d818064f02
```
Comment (ekelly55) [inline ios/Podfile.lock:704]: Based on the other lines here, should this be capitalized?

**Review by ekelly55 — APPROVED** [review]
Looks good to me with my limited React Native knowledge.

**Comment by ekelly55** [comment]
I reviewed and approved. Waiting for @hpweisberg to review before it can be merged.

File: App.tsx (hunk 1 above)
Comment (hpweisberg) [reply App.tsx:106]: Left there so others can see. Can be deleted if everyone has seen and understands.

File: ios/Podfile.lock (hunk 2 above)
Comment (hpweisberg) [reply ios/Podfile.lock:704]: It should not be, those are auto imported from yarn.

**Pull request merged**